src/index.html

```diff
@@ -17,11 +17,369 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.css"
     />
   </head>
-
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__img">
+        <img
+          src="./images/logo.png"
+          alt="logo Moyo"
+        />
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="index.html"
+              class="nav__link is-active"
+              data-qa="card"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
```

Comment:
dont forget to add empty lines between sibling elements , its better for code style